### DISCUSSION TITLE
New to DAX

### DISCUSSION THREAD

**Discussion by digitalagentTL** [DaxStudio/DaxStudio#752] [General]
Trying to analyze PowerBI report but keep getting this: No Databases were found when connecting to localhost:55503 (PowerBIDesktop) If your Power BI File is using a Live Connection please connect directly to the source model instead.
How do I resolve this issue?

**Comment by dgosbell** [comment]
That means that the report you have connected to in Power BI Desktop does not have any data model in it. DAX Studio needs to connect to the report that has the data in it. So if your PBIX file gets it's data from another file you need to open that other file and connect to that with DAX Studio
